docs-v2/guides/api-authorization/authorize-in-your-app-default-ui.mdx

```diff
@@ -120,6 +120,14 @@ Refer to the [Authorize in your app (custom UI)](/guides/api-authorization/autho
 
 ## 3. Listen for webhoooks & save the Connection ID (backend)
 
+<Accordion title="👩🏻‍💻 Simplified flow during development">
+  In development, you can retrieve the connection ID without relying on webhooks.
+
+	When using the `nango.openConnectUI()` method in the frontend SDK, the connection ID is available in the `event` parameter of the callback.
+
+	However, we do not recommend using this approach in production, as it is safer to avoid exposing the connection ID to the frontend.
+</Accordion>
```

Comment:
Should we a code snippet that would be clearer?
This is a snippet in `intro-to-nango`
```js
    await nango.openConnectUI({
        onEvent: (event) {
            if (event.type === 'connect') {
                // The auth flow succeeded. The user is connected!
                saveToDatabase(connectionId, integrationId);
            }
        }
    });
```

Reply:
Agreed!

Reply:
Done!